### PR TITLE
added updating robot software instructions to documentation

### PR DESCRIPTION
**Pull request by ablasdel**

### CODE REVIEW

File: source/updating.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+::
+
+    sudo apt-get update && sudo apt-get install --only-upgrade ros-indigo-*
+
+.. warning::
+
+    Using 'apt-get upgrade' could cause critical software on the robot, such
+    as the kernel, to change. Since we can not always guarantee your robot
+    will function after making such a change we suggest against using this command unless you have a good reason and understand the risks fully.
```
Comment (mikeferguson) [inline source/updating.rst]: long line, should be split

File: source/updating.rst [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Fetch Research Edition packages will be updated with new capabilities and bug
+fixes as needs become apparent.
+
+Setting up apt-get Sources
+--------------------
+
+The following commands will create files so that apt on the robot can see the
+public ros package server as well as the Fetch Robotics package server. You
+should only need to do this once, unless someone alters or deletes them.
```
Comment (mikeferguson) [inline source/updating.rst]: technically, they shouldn't have to do this at all -- it is already done when the robot ships. You should only have to run these lines if someone has changed settings.

**Comment by mikeferguson** [comment]
Some of this is already covered in the "Care and Feeding" section (https://github.com/fetchrobotics/docs/blob/master/source/care_and_feeding.rst#updating-your-robot) -- we should probably not duplicate the updating info. The "restore" info should probably also not be under "updating". I would suggest we create a separate "restore" section, and then put a note in the update section that "this depends on custom APT settings, if you have changed or damaged your APT setup, you may need to restore the settings" with a link to the restore section.

File: source/updating.rst (hunk 2 above)
Comment (ablasdel) [inline source/updating.rst]: True. I can make it "resetting up sources"

Comment (ablasdel) [inline source/updating.rst]: And "you shouldn't have to do this unless you great something"

Comment (ablasdel) [inline source/updating.rst]: Great-> break

Comment (mikeferguson) [inline source/updating.rst]: oh, I'm sure they thought what they were doing was a GREAT idea....

Comment (ablasdel) [inline source/updating.rst]: Did I tell you that VT chuong guy emailed me after the fact to thank me for fixing the robot he bricked? He said he was blocked from replying until then. haha

File: source/care_and_feeding.rst [hunk 3]
```diff
@@ -126,6 +126,13 @@ Upgrading to the latest packages is as easy as:
    sudo service robot stop
    sudo service robot start
 
+.. warning::
+
+    Using 'apt-get upgrade' could cause critical software on the robot, such
```
Comment (mikeferguson) [inline source/care_and_feeding.rst]: we should also call out dist-upgrade

File: source/care_and_feeding.rst [hunk 4]
```diff
@@ -133,6 +140,20 @@ be installed when the drivers are next started (typically by the robot upstart
...
+
+If someone has changed or deleted the default apt-get sources then the
+following commands will create sources.list files so that the robot can see
+the public ros package server and the Fetch Robotics package server.
+
+::
+
+    >$ sudo sh -c 'echo "deb http://packages.fetchrobotics.com/ros/ubuntu trusty main" > /etc/apt/sources.list.d/fetch-latest.list'
+
+    >$ sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu trusty main" > /etc/apt/sources.list.d/ros-latest.list'
```
Comment (mikeferguson) [inline source/care_and_feeding.rst]: this should be packages.fetchrobotics.com

**Pull request merged**